package-tarball.sh

```diff
@@ -7,54 +7,39 @@ set -v
 set -e
 
 # Set 'release' mode if requested, influences commitid.sh
-export RELEASE="0"
+export RELEASE="0"ß
 if [[ $# -gt 0 ]]
 then
     arg=$1
     shift
 
-    if [ "$arg" = "--release" ]
-    then
+    case $arg in
+    --release) 
         echo Saw $arg
         RELEASE="1"
-    else
+        ;;
+    *)
         echo "Unrecognized argument '$arg'"
         exit 1
-    fi
+        ;;
+    esac
```

Comment:
What's the point of changing this? It seems like the functionality is identical, is that right? Does this help avoid using autotools somehow?

Reply:
No, this isn't a functional change, but I think the argument parsing is more idiomatic. This will also make it easier to add arguments in the future.

Reply:
I'll undo this part of the change.